lib/datInfo.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+var bytes = require('prettier-bytes')
+
+function datPeers (dat) {
+  return dat.network ? dat.network.connected : 'N/A'
+}
+
+module.exports = {
+  datPeers,
```

Comment:
why not inline `datPeers` here too?

Reply:
`datPeers` is used in `datState`. If `datState` would use `this.datPeers` to access it the method could not be able to simply use it like: `var {datState} = require('./lib/datInfo')`

Reply:
+1